torch/__init__.py

```diff
@@ -267,10 +269,9 @@ def storage_type(cls):
     CharStorage, ByteStorage, HalfStorage
 }
 
-_tensor_classes = {
-    DoubleTensor, FloatTensor, LongTensor, IntTensor, ShortTensor,
-    CharTensor, ByteTensor, HalfTensor
-}
+# The _tensor_classes set is initialized by the call to _C._initialize_tensor_types()
+# which calls torch::tensor::initialize_python_bindings()
```

Comment:
I think just the first line is sufficient; someone can look at the implementation and see what it does, and this is likely to change and not be updated.